tests/integration/questionnaire/test_questionnaire_page_titles.py

```diff
@@ -6,17 +6,15 @@ def test_should_have_question_in_page_title_when_loading_introduction(self):
         # Given, When
         self.launchSurvey("test_final_confirmation")
         # Then
-        self.assertEqualPageTitle("Final confirmation to submit")
+        self.assertEqualPageTitle("None - Census 2021")
```

Comment:
We shouldn't ever print out 'None' as part of the page title?

Reply:
So, as we discussed: `{% set page_title = _(“Introduction”) %}` added and test renamed, thanks👍